### DISCUSSION TITLE
Define windows of time displayed

### DISCUSSION THREAD

**Discussion by feedhopper** [davish/obsidian-full-calendar#406] [Feature Requests]
Full calendar starts at midnight which is technically true for the start of day.

In practice, however, people will start working much later. It is of no use to display hours where people normally sleep.
As daily routine differ vastly (think nighshift), it would be best to let peple choose a timewindow to display.

An elegant solution would be to automatically adjust the timewindow in case of appointments outside of the normal time window.

An alternative solution could be to work with "zoom".  It would allow people to "zoom in" so far as to fit the needed time window to their needs and screen estate.

**Comment by davish** [comment]
Unfortunately, there's no way to set the minimum time available on a calendar in fullcalendar v5.0. Here is the setting from 4.0, but it has not been added to 5.0 yet: https://fullcalendar.io/docs/v4/minTime.

It is possible to set the default scroll time, which is now set to 6am: https://fullcalendar.io/docs/scrollTime

**Comment by feedhopper** [comment]
Oh, wow. First of all: THANK YOU!
:)

I wasn't aware of the first issue you mentioned.

How can I achieve the scroll time workaround in Obsidian?

**Comment by davish** [comment]
It hasn't been added to the plugin settings yet, but it is possible. This issue will be closed when it's implemented.

**Comment by feedhopper** [comment]
Ahhh. THANK YOU ONCE AGAIN!
